src/main/java/com/alkemy/ong/auth/controller/UserAuthController.java

```diff
@@ -1,16 +1,21 @@
 package com.alkemy.ong.auth.controller;
 
+import com.alkemy.ong.auth.domain.UserDomain;
+import com.alkemy.ong.auth.dto.UserDTO;
```

Comment:
I also added a UserDTO you can import, my question is , should UserDTO be at the auth package? or in the source package? let me know what is best

Reply:
I think it is indistinct but there may be other options